00004_well_adding_payments.js

```diff
@@ -0,0 +1,93 @@
+const Common = require("../../../../setup/common.js");
+
+const folderName = __dirname.match(/[^\\/]+$/);
+
+Cypress.on("uncaught:exception", (err) => {
+   if (
+      err.message.includes(
+         "TypeError: Cannot read properties of undefined (reading 'raw')"
+      )
+   ) {
+      return false;
+   }
+});
+
+describe.only("Adding Payments", () => {
```

Comment:
can remove `only`, not a big deal since this is the only one the file, but better in case we want to add in the future.

Reply:
I removed .only and update definition.